flake-module.nix

```diff
@@ -50,12 +50,35 @@ in
                 '';
               };
               port = mkOption {
-                type = types.int;
-                default = 8080;
+                type = types.nullOr types.int;
+                default = null;
                 description = ''
                   Port to serve process-compose's Swagger API on.
                 '';
               };
+              tui = mkOption {
+                type = types.nullOr types.bool;
+                default = null;
+                description = "Enable or disable the TUI for the application.";
+              };
+              extraCliArgs =
+                let
+                  cliArgsAttr = {
+                    port = "-p ${toString config.process-compose.port}";
+                    tui = "-t=${lib.boolToString config.process-compose.tui}";
+                  };
+                  getCliArgs =
```

Comment:
Naming is problematic here. 

- `getCliArgs` == a verb (not a noun) that indicates that it is a function that will return cliArgs (whatever that is)
- In reality, this variable holds a portion of cli argument string (thus, a noun)